CONTRIBUTING.md

```diff
@@ -96,7 +96,9 @@ few prerequisite steps:
 * Make sure you have a [GitHub account](https://github.com/signup/free)
 * Make sure you have [set up git locally](https://help.github.com/articles/set-up-git)
 * On Windows, run `git config --global core.longpaths true`
-so that you are able to clone the Hibernate Search repository
+so that you are able to clone the Hibernate Search repository;
+also run `git config --global core.autocrlf input` to make sure that line ending characters are kept unchanged 
```

Comment:
Maybe we should add a `.gitattributes` file instead?

https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#per-repository-settings

Reply:
That could work as well 😃

Reply:
We already have one :/

Can you tell me which files you had trouble with exactly?

Reply:
I added a commit to my branch, to handle this as discussed on Zulip.